azurepipelines/e2e_test/scenarios/test_runner/Bundle-update.py

```diff
@@ -49,6 +50,16 @@ def test_AptDeployment(self):
         self.aduTestConfig = DuAutomatedTestConfigurationManager.FromOSEnvironment()
         self.duTestHelper = self.aduTestConfig.CreateDeviceUpdateTestHelper()
 
+        self.aduScenarioDefinition = DuScenarioDefinitionManager.FromOSEnvironment()
+
+        test_device_id = self.aduScenarioDefinition.test_device_id
+        test_adu_group = self.aduScenarioDefinition.test_adu_group
+        global test_result_file_prefix
+        test_result_file_prefix = self.aduScenarioDefinition.test_result_file_prefix
+        test_bundle_update_deployment_id = self.aduScenarioDefinition.test_bundle_update_deployment_id
+        test_connection_timeout_tries = self.aduScenarioDefinition.test_connection_timeout_tries
+        retry_wait_time_in_seconds = self.aduScenarioDefinition.retry_wait_time_in_seconds
+
```

Comment:
The below commits don't look like they track from the latest merge to develop (completed Monday). Is there any chance we're missing those commits?